README.md

```diff
@@ -54,19 +31,27 @@ Each script supports the `--help` or `-h` command line parameter, which will lis
 * `run_plugin.js`: executes a plugin via a direct MongoDB connection.
 * `merge.js`: merges two branches if there are no conflicts.
 * `usermanager.js`: manages users, organizations, and project authorization (read, write, delete).
+* `clean_up.js`: lists/removes projects based on supplied criteria (commits, branches, regex etc.).
 * `export.js`: exports a (snapshot of a) branch into a json file.
 * `import.js`: imports a (snapshot of a) branch into a webgme project.
-* `parse_xme.js` __outdated__: parses a desktop GME xme file and imports it into a webgme database.
-* `serialize_to_xml.js` __outdated__: creates a desktop GME xme file from a given webgme project.
 
-# Extensions
+#Extension Components
 * [Plugins](./src/plugin/README.md) - Model interpretation for e.g. code generation.
 * [AddOns](./src/addon/README.md) - Continuous model interpretation for e.g. constraint evaluation.
 * [Executor](./src/server/middleware/executor/Readme.md) - Code execution framework.
+* [Rest Routers](./src/server/middleware/ExampleRestRouter.js) - Add custom REST API routes.
+* [Layouts](./src/plugin/coreplugins/LayoutGenerator/LayoutGenerator.js) - Configure the layout of the generic UI.
+* [Visualizers](./src/plugin/coreplugins/VisualizerGenerator/VisualizerGenerator.js) - Add complete visualizers to the generic UI.
+* [Decorators](./src/plugin/coreplugins/DecoratorGenerator/DecoratorGenerator.js) - Add custom decoration to the nodes in the model editor.
+* [Constraints](./src/plugin/coreplugins/ConstraintEvaluator/ConstraintEvaluator.js) - Add custom constraints based on meta-types.
 
-# Contributing
-See [CONTRIBUTING.md](./CONTRIBUTING.md)
+See [gme-config](./config/README.md) for avaiable configuration parameters.
 
-# License
+#Change log
+See [CHANGELOG.md](./CHANGELOG.md)
+
+#Contributing
+See [CONTRIBUTING.md](./CONTRIBUTING.md)
 
+#License
 See the [LICENSE](LICENSE) file.
```

Comment:
Does this link work? 

[LICENSE](./LICENSE) or should we change the contributing/change log links?